tools/BuildAssets/psModules/CodeGenerationModules/AutoRestCodeGenerationModule/AutoRestCodeGenerationModule.psm1

```diff
@@ -147,6 +150,11 @@ param(
         $args = $args + " --tag=$ConfigFileTag"
     }
 
+    if($ApiType -eq "multi-api")
```

Comment:
@dsgouda also make sure the default option should be SignleApi (without user specifying it)
Only when someone wants to generate MultiApi package, they should provide the parameter.